src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -38,6 +43,7 @@ export class DotEditPageNavComponent implements OnInit {
                 return this.dotLicenseService.isEnterpriseLicense();
             })
             .mergeMap((isEnterpriseLicense: boolean) => {
+                this.isEnterpriseLicense = isEnterpriseLicense;
                 return Observable.of(this.getNavItems(this.pageState, isEnterpriseLicense));
```

Comment:
I just notice this but, you don't need the ```Observable.of``` if instead of ```mergeMap``` you use ```map```.

PD. Use pipes.